src/content/docs/fr/guides/integrations-guide/tailwind.mdx

```diff
@@ -195,34 +244,26 @@ export default {
 };
 ```
 
-### Les modificateurs basés sur les classes ne fonctionnent pas avec les directives  `@apply`
+### Class-based modifiers do not work with `@apply` directives
```

Comment:
Any particular reason this is translated back to english ?

Reply:
A translation oversight, I'm changing it! Thanks